.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches:
+      - development
+      - base
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    environment:
+      name: development
+
+    env:
+      APP_NAME: digitized_av_validation
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4.0.2
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          role-to-assume: ${{ secrets.AWS_DEPLOY_ROLE }}
+          role-skip-session-tagging: true
+          role-duration-seconds: 1000
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: ECR login
+        run: aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.us-east-1.amazonaws.com
+
+      - name: Set environment variables
+        run: |
+          echo "DATE=$(date +'%Y%m%dT%H%M%S')" >> $GITHUB_ENV
+
+      - name: Build Docker image
+        run: docker build -t ${{ env.APP_NAME }} --target build .
+
+      - name: Tag and push latest to ECR
```

Comment:
I think there's an open question here about whether we want the `latest` to be created for builds in both `development` and `base` or just `base`. Because right now it looks like it's both and I'm not sure about that. Can you look at best practices around that?

Reply:
Also, all of these tags and pushes might be a good argument for us building our own hook that does this. Maintaining this in multiple apps feels like a bear.

Reply:
Looks like best practice is to only use latest for testing purposes. According to AWS documentation [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/container-considerations.html#:~:text=We%20recommend%20that%20you%20use,container%20image%20for%20every%20commit.).

> Container images are stored in a container registry. Each image in a registry is identified by a tag. There's a tag called latest. This tag functions as a pointer to the latest version of the application container image, similar to the HEAD in a git repository. We recommend that you use the latest tag only for testing purposes. As a best practice, tag container images with a unique tag for each build. We recommend that you tag your images using the git SHA for the git commit that was used to build the image.

Reply:
I'm going to update to include an update to latest only on the base branch

Reply:
OK let's talk some more about this. I'm wondering if we should be building at all in base, or if we should just be retagging.